### PR TITLE
fix(Workbench): Add description to Workbench.Header

### PR DESCRIPTION
**Pull request by wichniowski**
The description was missing in the Workbench.Header component. This PR will add it.

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Workbench/Workbench.tsx [hunk 1]
```diff
@@ -2,11 +2,13 @@ import React, { Component } from 'react';
 import cn from 'classnames';
 import Heading from '../Typography/Heading';
 import IconButton from '../IconButton';
+import Paragraph from '../Typography/Paragraph';
```
Comment (m10l) [inline packages/forma-36-react-components/src/components/Workbench/Workbench.tsx]: Maybe a bit nitpicky so feel free to ignore, but I'm not sure if this makes sense as a `Paragraph` component. I'd use `Paragraphs` for longer strings of (normally multiline) text.

File: packages/forma-36-react-components/src/components/Workbench/Workbench.css [hunk 2]
```diff
@@ -65,6 +65,14 @@
   flex-direction: row;
 }
 
+.Workbench__header-description {
+  margin-left: var(--spacing-m);
+}
+
+.Workbench__header-description__text {
+  margin-left: var(--color-element-light);
```
Comment (m10l) [inline packages/forma-36-react-components/src/components/Workbench/Workbench.css]: Typo? I guess you meant `color` here.

**Comment by wichniowski** [comment]
@m10l I updated the PR. It now also handles overflowing titles and descriptions

File: packages/forma-36-react-components/src/components/Workbench/Workbench.css [hunk 3]
```diff
@@ -63,6 +63,27 @@
   display: flex;
   align-items: center;
   flex-direction: row;
+  overflow: hidden;
+  flex: 0 1 auto;
+}
+
+.Workbench__header-title h1 {
```
Comment (m10l) [inline packages/forma-36-react-components/src/components/Workbench/Workbench.css]: Please no element selectors

Comment (wichniowski) [reply packages/forma-36-react-components/src/components/Workbench/Workbench.css]: @m10l done

**Pull request merged**